x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java

```diff
@@ -1230,8 +1229,8 @@ private void assertHistoryIsPresent(
             fail("failed to perform search:" + e.getMessage());
         }
 
-        // Finally, check that the history index is in a good state
```

Comment:
Nit: is there a reason you removed this comment? I don't think it should get an aware for "most valuable comment ever", but I do think it helps a little bit.